src/main/java/com/parasoft/demoapp/controller/PageController.java

```diff
@@ -143,4 +143,15 @@ public String showAccessDeniedPage(ModelMap modelMap) {
         }
         return "error/403";
     }
+
+    @GetMapping("/conflict")
+    public String showConflictPage(ModelMap modelMap, String type) {
+        try {
+            modelMap.addAttribute("conflictType", type);
+            modelMap.addAttribute("currentWebServiceMode", globalPreferencesService.getCurrentGlobalPreferences().getWebServiceMode().getValue());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please pull the latest merged code. Let's replace all new use of `printStackTrace `to be error logging.